aspnetcore/tutorials/publish-to-azure-api-management-using-vs.md

```diff
@@ -4,192 +4,147 @@ author: codemillmatt
+1. When that completes, select the **Finish** button.
 1. The dialog closes and a summary screen appears with information about the publish. Select the **Publish** button.
 
-    ![Screenshot of Visual Studio with the publish profile displayed](publish-to-azure-api-management-using-vs/_static/vs_publish_profile.png)
-
     The web API will publish to both Azure App Service and Azure API Management. A new browser window will appear and show the API running in Azure App Service. You can close that window.
 
-1. Switch back to the Azure API Management instance in the Azure portal.
-1. Refresh the browser window.
-1. Select the blank API you created in the preceding steps. It's now populated and you can explore around.
-
-    ![Screenshot of the populated API in Azure API Management](publish-to-azure-api-management-using-vs/_static/deployed_to_azure_api_mgmt.png)
+1. Open up the Azure portal in a web browser and navigate to the API Management instance you created.
+1. Select the **APIs** option from the left-hand menu.
+1. Select the API you created in the preceding steps. It's now populated and you can explore around.
 
 ### Configure the published API name
 
-Notice the name of the API is different than what you named it. The published API is named *WeatherAPI*; however, you named it *WeatherForecasts* when you created it. Complete the following steps to fix the name:
+Notice the name of the API is named *WeatherAPI*; however, we would like to call it *Weather Forecasts*. Complete the following steps to update the name:
 
-![Screenshot of populated API with different names highlighted](publish-to-azure-api-management-using-vs/_static/deployed_to_azure_api_mgmt_wrong_name.png)
-
-1. Delete the following line from the `Startup.ConfigureServices` method:
-    
-    ```csharp
-    services.AddSwaggerGen();
-    ```
-
-1. Add the following code to the `Startup.ConfigureServices` method:
+1. Add the following to `Program.cs` immediately after `servies.AddSwaggerGen();`
     
     ```csharp
-    services.AddSwaggerGen(config =>
+    builder.Services.ConfigureSwaggerGen(setup =>
     {
-        config.SwaggerDoc("v1", new Microsoft.OpenApi.Models.OpenApiInfo
+        setup.SwaggerDoc("v1", new Microsoft.OpenApi.Models.OpenApiInfo
         {
-            Title = "WeatherForecasts",
+            Title = "Weather Forecasts",
             Version = "v1"
         });
     });
     ```
-
-1. Open the newly created publish profile. It can be found from **Solution Explorer** in the *Properties/PublishProfiles* folder.
-
-    ![Screenshot showing the publish profile file location highlighted](publish-to-azure-api-management-using-vs/_static/vs_publish_profile_highlighted.png)
-
-1. Change the `<OpenAPIDocumentName>` element's value from `v1` to `WeatherForecasts`.
-
-    ![screenshot of the changes necessary for the publish profile](publish-to-azure-api-management-using-vs/_static/vs_publish_profile_changes.png)
-
+   
```

Comment:
All of this is really clumsy. Can you open an issue on this doc and I'll create the project and import snippets. With snippets you can show more code and highlight the updates, changes, etc.

Reply:
Yes, I'll create the issue after the merge.